cmd/konvoy-image/cmd/create-package-bundle.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	packageCmd        = "create-package-bundle"
+	packageCmdExample = "create-package-bundle --os redhat-8.4 --kubernetes-version=1.28.6 --output-directory=artifacts"
+	validOS           = []string{
+		"centos-7.9",
+		"redhat-7.9",
+		"redhat-8.4",
+		"redhat-8.6",
+		"redhat-8.8",
+		"oracle-7.9",
+		"rocky-9.1",
+		"ubuntu-18.04",
+		"ubuntu-20.04",
+	}
+)
+
+type packageBundleCmdFlags struct {
+	targetOS          string
+	kubernetesVersion string
+	fips              bool
+	outputDirectory   string
+	containerImage    string
+}
+
+var packageBundleFlags packageBundleCmdFlags
+
+// NOTE Usage information always comes from the konvoy-image command, not the konvoy-image-wrapper command.
+// We define this command only to show usage; it does nothing else.
```

Comment:
Remind me again please why there is the 2 Commands?

Reply:
so that we can get the usage text